src/EPR.Payment.Service/HealthCheck/PaymentStatusHealthCheck.cs

```diff
@@ -0,0 +1,25 @@
+﻿using EPR.Payment.Service.Services.Interfaces;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
```

Comment:
Replace the file with the one I had sent to include cancellationtokens

Reply:
resolved